logging/logging_alerts/custom-logging-alerts.adoc

```diff
@@ -0,0 +1,40 @@
+:_content-type: ASSEMBLY
+[id="custom-logging-alerts"]
+include::_attributes/common-attributes.adoc[]
+= Custom logging alerts
+:context: custom-logging-alerts
+
+toc::[]
+
+In logging 5.7 and later versions, users can configure the LokiStack deployment to produce customized alerts and recorded metrics. If you want to use customized link:https://grafana.com/docs/loki/latest/alert/[alerting and recording rules], you must enable the LokiStack ruler component.
+
+LokiStack log-based alerts and recorded metrics are triggered by providing link:https://grafana.com/docs/loki/latest/query/[LogQL] expressions to the ruler component. The Loki Operator manages a ruler that is optimized for the selected LokiStack size, which can be 1x.extra-small, 1x.small, or 1x.medium.
+
+[NOTE]
+====
+1x.extra-small is for demo purposes only, and is not supported.
+====
+
+To provide these expressions, you must create an `AlertingRule` custom resource (CR) or a `RecordingRule` custom resource (CR), containing Prometheus-compatible link:https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/[alerting rules] or link:https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/[recording rules], respectively.
+
+Administrators can configure log-based alerts or recorded metrics for `application`, `audit`, or `infrastructure` tenants. Users without administrator permissions can configure log-based alerts or recorded metrics for `application` tenants of the applications that they have access to.
+
+Application, audit, and infrastructure alerts are sent by default to the {product-title} monitoring stack Alertmanager in the `openshift-monitoring` namespace, unless you have disabled the local Alertmanager instance. However, if the Alertmanager that is used to monitor user-defined projects in the `openshift-user-workload-monitoring` namespace is enabled, application alerts are sent here by default.
```

Comment:
Sort of. It's a specific Alertmanager that's used to monitor user workloads, AFAIK.

Maybe:
"If the Alertmanager that is used to monitor user-defined projects in the `openshift-user-workload-monitoring` namespace is enabled, application alerts are sent to this namespace by default."

I'd probably check with @JoaoBraveCoding before making any changes here

Reply:
> Sort of. It's a specific Alertmanager that's used to monitor user workloads, AFAIK.

Correct 👌 

> "If the Alertmanager that is used to monitor user-defined projects in the openshift-user-workload-monitoring namespace is enabled, application alerts are sent to this namespace by default."

This is also fine I'm just not sure about the end part "application alerts are sent to this namespace by default." since alerts are sent to the Alertmanager running in this namespace rather than the namespace itself, but I realize I'm being picky. I'll let you decide :)

Reply:
Going with "If the Alertmanager that is used to monitor user-defined projects in the `openshift-user-workload-monitoring` namespace is enabled, application alerts are sent to the Alertmanager in this namespace by default." 🙂 thank you!